roles/geotaste_setup/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
it doesn't look like you have any app-specific setup needed? so this role can be removed